### PR TITLE
docker-ls 0.2.0 (new formula)

### PR DESCRIPTION
**Pull request by eppsilon**
- [X] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md) document?
- [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [X] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

File: Formula/docker-ls.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    ENV["AUTO_GOPATH"] = "1"
+
+    system "make"
+
+    bin.install "build/bin/docker-ls" => "docker-ls"
+    bin.install "build/bin/docker-rm" => "docker-rm"
+  end
+
+  test do
+    system "#{bin}/docker-ls", "version"
```
Comment (MikeMcQuaid) [inline Formula/docker-ls.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

File: Formula/docker-ls.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+class DockerLs < Formula
+  desc "CLI tools for browsing and manipulating Docker registries"
+  homepage "https://github.com/mayflower/docker-ls"
+  url "https://github.com/mayflower/docker-ls.git",
+      :tag => "v0.2.0",
+      :revision => "45b56b14f05d2e2611a4d6bce7555e8ec04b0c2c"
```
Comment (bfontaine) [inline Formula/docker-ls.rb]: Do we really need the tag here or does it build with the tarball?

Comment (MikeMcQuaid) [inline Formula/docker-ls.rb]: Please use the tarball unless it's broken without.

**Comment by MikeMcQuaid** [comment]
@eppsilon Any news on the comments above? Thanks!

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by woodruffw** [comment]
Closing due to inactivity.

**Pull request closed without merging**